src/components/tx/modals/TokenTransferModal/useSafeTokenWarning.ts

```diff
@@ -0,0 +1,30 @@
+import useChainId from '@/hooks/useChainId'
+import { getSafeTokenAddress } from '@/components/common/SafeTokenWidget'
+import { useWeb3ReadOnly } from '@/hooks/wallets/web3'
+import useAsync from '@/hooks/useAsync'
+import { Contract } from 'ethers'
+import { Interface } from '@ethersproject/abi'
+
+// TODO: Remove this hook after the safe token has been unpaused
+const useSafeTokenWarning = () => {
```

Comment:
Nit: I'd rename this to `useIsSafeTokenPaused` and return a `boolean`. I was expecting this to return the string, which we could alternatively do of course.